cpp/src/arrow/util/decimal-benchmark.cc

```diff
@@ -44,7 +44,46 @@ static void FromString(benchmark::State& state) {  // NOLINT non-const reference
   state.SetItemsProcessed(state.iterations() * values.size());
 }
 
+static void BinaryLogicalOp(benchmark::State& state) {  // NOLINT non-const reference
+  BasicDecimal128 d1(546, 123), d2(123, 456);
+  while (state.KeepRunning()) {
+    benchmark::DoNotOptimize(d1 == d2);
+    benchmark::DoNotOptimize(d1 <= d2);
+    benchmark::DoNotOptimize(d1 >= d2);
+    benchmark::DoNotOptimize(d1 >= d1);
+  }
+}
+
+static void BinaryMathOp(benchmark::State& state) {  // NOLINT non-const reference
+  BasicDecimal128 d1(546, 123), d2(123, 456);
+  while (state.KeepRunning()) {
+    benchmark::DoNotOptimize(d1 - d2);
+    benchmark::DoNotOptimize(d1 + d2);
+    benchmark::DoNotOptimize(d1 * d2);
```

Comment:
can you please add div/mod also ?

Reply:
will do.  Note I didn't update these method because the benchmark showed const ref was the same or potentially better then by value.  I thought it might have something to do with level of complexity of the function being called (or just noise)